assets/src/domain/shared/entities/datetimes/predicates/isExpired.ts

```diff
@@ -0,0 +1,16 @@
+import { is } from 'ramda';
+import { parseISO } from 'date-fns';
+
+import { Datetime } from '../../../../eventEditor/services/apollo/types';
+import { diff } from '../../../../../application/services/utilities/date';
+import { isValidOrTrashed } from '../../../services/predicates';
+import { now } from './filters';
+
+const isExpired = (date: Datetime): boolean => {
+	return (
+		(isValidOrTrashed(date) && is(Boolean, date.isExpired) && date.isExpired) ||
```

Comment:
I don't think `is(Boolean, date.isExpired)` check is necessary.

Reply:
Yes, that's a bit too much safety, updated.